components/list/list-item-nav-button-mixin.js

```diff
@@ -18,32 +18,41 @@ export const ListItemNavButtonMixin = superclass => class extends ListItemButton
 	static get styles() {
 
 		const styles = [ css`
+			:host {
+				--d2l-list-item-border-color: var(--d2l-color-mica) !important; /* clean up with flag GAUD-7495-list-item-new-styles */
+			}
 			:host(:not([button-disabled])) {
 				--d2l-list-item-content-text-color: var(--d2l-color-ferrite);
 			}
 			.d2l-list-item-content ::slotted(*) {
 				width: 100%;
 			}
 			:host([current]) [slot="outside-control-container"] {
+				--d2l-list-item-border-color: var(--d2l-color-celestine);
+				background-color: var(--d2l-color-regolith);
 				border: 3px solid var(--d2l-color-celestine);
 				margin-block: -1px;
 			}
-			:host([_focusing-primary-action]:not([current])) [slot="outside-control-container"] {
-				border: 2px solid var(--d2l-color-celestine);
-			}
 			:host([current]) [slot="control-container"]::before,
 			:host([current]) [slot="control-container"]::after {
 				border-color: transparent;
 			}
-			:host([_focusing-primary-action]) .d2l-list-item-content {
-				--d2l-list-item-content-text-outline: none;
-			}
 			:host([_hovering-primary-action]) .d2l-list-item-content,
 			:host([_focusing-primary-action]) .d2l-list-item-content {
 				--d2l-list-item-content-text-color: var(--d2l-color-ferrite);
 				--d2l-list-item-content-text-decoration: none;
 			}
 
+			/* clean up below with flag GAUD-7495-list-item-new-styles */
+			:host(:not([selection-disabled]):not([button-disabled]):not([skeleton])[_focusing]) [slot="outside-control-container"] {
+				border-color: var(--d2l-list-item-border-color);
+				margin-bottom: -1px;
+			}
+			@media only screen and (hover: hover), only screen and (pointer: fine) {
+				:host([_focusing]) d2l-list-item-drag-handle {
+					opacity: 1;
+				}
+			}
```

Comment:
We're applying these styles from the start to list-item-nav-button-mixin usages.